reference-architecture/day2ops/scripts/ocp36-sat6.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# vim: sw=4 ts=4 et
+import os, argparse, socket, getpass, subprocess
+
+class ocpSat6(object):
+
+    __name__ = 'ocpSat6'
+
+    openshift3Images=[]
+
+    def __init__(self, load=True):
+
+        if load:
+            self._parseCli()
+            self._loadImageList()
+            self._addData()
+            self._syncData()
+
+    def _loadImageList(self):
+        cmd='curl -s https://registry.access.redhat.com/v1/search?q="openshift3" | python -mjson.tool | grep ".name.:" | cut -d: -f2 | sed -e "s/ "//g"" -e "s/,"//g""'
+        result = subprocess.check_output(cmd, shell=True)
+        lines = result.splitlines()
+        for line in lines:
+            nl = line.replace('"', '')
+            self.openshift3Images.append(nl)
+
+    def _parseCli(self):
+
+        parser = argparse.ArgumentParser(description='Add all OCP images for disconnected installation to satellite 6', add_help=True)
+        parser.add_argument('--orgid', action='store', default='1',help='Satellite organization ID to create new product for OCP images in')
+        parser.add_argument('--productname', action='store', default='ocp36',help='Satellite product name to use to create OCP images')
+        parser.add_argument('--password', action='store', help='Password for hammer CLI')
+        self.args = parser.parse_args()
+
+        if not self.args.password:
+            self.args.password = getpass.getpass(prompt='Please enter the password to use for the admin account in hammer CLI: ')
+
+    def _syncData(self):
+        print "Sync repo data? (This may take a while)"
+        go = raw_input("Continue? y/n:\n")
```

Comment:
A "-y" flag can be also helpful to run it without asking anything to the user.

Reply:
Sure. That works